src/ChangesReporting/Output/ConsoleOutputFormatter.php

```diff
@@ -83,15 +88,42 @@ private function reportFileDiffs(array $fileDiffs, bool $absoluteFilePath): void
                 $filePath .= ':' . $firstLineNumber;
             }
 
-            $message = sprintf('<options=bold>%d) %s</>', ++$i, $filePath);
+            $filePathWithUrl = $this->addEditorUrl(
+                $filePath,
+                $fileDiff->getAbsoluteFilePath(),
+                $fileDiff->getRelativeFilePath(),
+                (string) $fileDiff->getFirstLineNumber()
+            );
+
+            $message = sprintf('<options=bold>%d) %s</>', ++$i, $filePathWithUrl);
 
             $this->symfonyStyle->writeln($message);
             $this->symfonyStyle->newLine();
             $this->symfonyStyle->writeln($fileDiff->getDiffConsoleFormatted());
 
             if ($fileDiff->getRectorChanges() !== []) {
+                $relativePathHelper = new ParentDirectoryRelativePathHelper(getcwd());
                 $this->symfonyStyle->writeln('<options=underscore>Applied rules:</>');
-                $this->symfonyStyle->listing($fileDiff->getRectorShortClasses());
+                $appliedRules = [];
+                foreach ($fileDiff->getRectorClasses() as $rectorClass) {
+                    $appliedRuleName = (string) Strings::after($rectorClass, '\\', -1);
```

Comment:
For the rules applied, we need to get the absolute URL of the corresponding rule file, we can do that using reflection. Then we can use the `ParentDirectoryRelativePathHelper` tool from PHPStan to get the relative URL